clients/html/src/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <base href="/get-a-quote"/>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/x-icon" href="assets/favicon.png">
+    <link rel="icon" type="image/x-icon" href="assets/favicon.png?v=2">

```

Comment:
curious, why `?v=2` here ?

Reply:
it forces browsers to download a new version initially - https://stackoverflow.com/questions/2208933/how-do-i-force-a-favicon-refresh

Reply:
ah. since, this is going to be the first version, we probably don't need it. (we don't have a production site yet, so no users who's using it currently)